client.go

```diff
@@ -42,6 +43,14 @@ type Client interface {
 	// offset, OffsetNewest for the offset of the message that will be produced next, or a time.
 	GetOffset(topic string, partitionID int32, time int64) (int64, error)
 
+	// Coordinator returns the coordinating broker for a consumer group. It will return a locally cached
+	// value if it's available. This may be stale, in which case you should call RefreshCoordinator
+	// to update the locally cached value.
+	Coordinator(consumerGroup string) (*Broker, error)
+
+	// RefreshCoordinator refreshes the coordinator for a consumer group.
```

Comment:
mention that "refreshing the coordinator" involves asking the cluster, not just some local operation